src/prognostic_equations/edmfx_closures.jl

```diff
@@ -9,7 +9,7 @@ import ClimaCore.Fields as Fields
 """
     Return buoyancy on cell centers.
 """
-function ᶜbuoyancy(params, ᶜρ_ref::FT, ᶜρ::FT) where {FT}
+function ᶜphysical_buoyancy(params, ᶜρ_ref::FT, ᶜρ::FT) where {FT}
```

Comment:
@trontrytel I change the name to `ᶜphysical_buoyancy` as I find it a bit confusing that `ᶜbuoyancy` returns physical buoyancy, but `ᶠbuoyancy` returns covariant3 buoyancy